frontend/src/store/playerElement.ts

```diff
@@ -94,13 +94,20 @@ class PlayerElementStore {
       jassub = new JASSUB({
         video: mediaElementRef.value,
         subUrl: trackSrc,
+        fonts: loadedFonts,
         workerUrl: jassubWorker,
         availableFonts: { 'liberation sans': jassubDefaultFont },
         // Both parameters needed for subs to work on iOS
         prescaleFactor: 0.8,
         onDemandRender: false
       });
     } else if (jassub) {
+      if (Array.isArray(loadedFonts)) {
+        for (const font of loadedFonts) {
+          jassub.addFont(font);
+        }
+      }
+
```

Comment:
```suggestion
    } else if (jassub && Array.isArray(loadedFonts)) {
        for (const font of loadedFonts) {
          jassub.addFont(font);
        }
```

Also, are you sure this is ever going to be needed? I'm on mobile right now, but iirc we created/destroyed jassub every time the currently playing item changes? Or maybe just on unmount?